### PR TITLE
create bookmark html

### PR DESCRIPTION
**Pull request by IsabellUrban**

### CODE REVIEW

File: bookmark.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                src="assets/home_app_logo_FILL0_wght400_GRAD0_opsz24.svg"
+                alt="Home grafik"
+            /></a>
+          </li>
+          <li>
+            <a href="./bookmark.html">
+              <img
+                class="image image-select"
+                src="assets/offline_bolt_FILL0_wght400_GRAD0_opsz24.svg"
+                alt="Bookmark grafik"
```
Comment (eneben) [inline bookmark.html:64]: Is the last bookmark not selected here? I did this in such a way that only those cards appear in the bookmarks.html that also have a marked bookmark in the index.html. And the bookmarks.html can only contain cards with marked bookmarks, that's the logic behind it.

**Review by eneben — APPROVED** [review]
I think you did great. Just a small logic question, but it doesn't have anything to do with the code, that's why I gave you a green tick.

**Comment by eneben** [comment]
The vercel link still needs a request, maybe you can change that.

File: bookmark.html (hunk 1 above)
Comment (IsabellUrban) [reply bookmark.html:64]: Yes, you're right. I changed it. :-)

**Pull request merged**